ZUGFeRD/Tax.cs

```diff
@@ -33,13 +33,12 @@ public class Tax
         /// 
         /// This information is calculated live.
         /// </summary>
-        public decimal TaxAmount
-        {
-            get
-            {
-                return 0.01m * this.Percent * this.BasisAmount;
-            }
-        }
+        public decimal TaxAmount => TaxAmountCalculated ?? 0.01m * Percent * BasisAmount;
+
+        /// <summary>
+        /// Overrides TaxAmount in case calculated by caller
+        /// </summary>
+        public decimal? TaxAmountCalculated { get; set; }

```

Comment:
wouldn't it make more sense to make this property private or at least internal since it is not needed by any outside caller?

Can you please also add unit tests? If you need assistance, please let me know!